arches_her/templates/views/components/reports/scenes/archive.htm

```diff
@@ -6,6 +6,7 @@
 
 <!-- Bibliographic Source Creation section -->
 <div class="aher-report-section" data-bind="visible: !!dataConfig.sourceCreation">
+    <p class="sr-only" data-bind="text: document.querySelector('.aher-report-anchors li.active').innerText + ' tab now displayed'"></p>
```

Comment:
@phudson-he  - can you wrap "tab now displayed" in a `{% trans  "" %}` tag

something like...
```
<p class="sr-only" data-bind="text: document.querySelector('.aher-report-anchors li.active').innerText + '{% trans " tab now displayed" %}'"></p>
```

Reply:
Added the trans tag in commit https://github.com/archesproject/arches-her/pull/1303/commits/b23b9f074d7590189d0c9956dcc1f5dccbad7153